arm_robot_1:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_link_1
    pid: {p: 30, i: 5, d: 0.3}

  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_link_2
    pid: {p: 30, i: 5, d: 0.3}

  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_link_3
    pid: {p: 30, i: 5, d: 0.3}

  joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_link_4
    pid: {p: 30 , i: 5, d: 0.3}